example/lib/operations.dart

```diff
@@ -227,7 +244,8 @@ runOperations() async {
     print('Here is our mnemonic: \n$mnemonic');
 
     String dondo =
-        "imitate embody law mammal exotic transfer roof hope price swift ordinary uncle";
+        "able student evoke travel find shift gasp beauty venture dove valid lounge";
```

Comment:
Please remove it.

Reply:
too late, lol, by now all her eth and tron have been transfered by someone.

Reply:
@Rakeally keep in mind these mnemonics are forever compromised. Never put serious money in it.